### PR TITLE
Pass ssh flags to rsync

### LINKED ISSUE #19: Non standard ssh port not working
Tried to specify a different port for ssh on my server  
`sshcode user@server.io:port` returns error
`sshcode user@server.io  --ssh-flags '-p xxxx' ` is ignored

How should it be done?

### PR DESCRIPTION
**Pull request by ammario**
Resolves #19

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -77,14 +80,14 @@ chmod +x `+codeServerPath+`
...
-		err = syncUserSettings(host)
+		err = syncUserSettings(*sshFlags, host)
 		if err != nil {
 			flog.Fatal("failed to sync settings: %v", err)
 		}
 		flog.Info("synced settings in %s", time.Since(start))
 
 		flog.Info("syncing extensions")
-		err = syncExtensions(host)
+		err = syncExtensions(host, *sshFlags)
```
Comment (scsmithr) [inline main.go]: Args should be flipped.

File: main.go [hunk 2]
```diff
@@ -53,22 +54,24 @@ More info: https://github.com/codercom/sshcode
...
 	const codeServerPath = "/tmp/codessh-code-server"
 
-	// Downloads the latest code-server and allows it to be executed.
-	sshCmd := exec.Command("ssh",
-		"-tt",
-		host,
-		`/bin/bash -c 'set -euxo pipefail || exit 1
-wget -q https://codesrv-ci.cdr.sh/latest-linux -O `+codeServerPath+`
+	downloadScript := `/bin/bash -c 'set -euxo pipefail || exit 1
+wget -q https://codesrv-ci.cdr.sh/latest-linux -O ` + codeServerPath + `
```
Comment (nhooyr) [inline main.go]: why two wget?

Comment (ammario) [reply main.go]: No idea how that got there.. That script is completely broken. We're going to want some tests

Comment (nhooyr) [reply main.go]: I think its a good idea we http.Get on our side and then just write the binary to the remote side if it has changed. Would be nice and easy to test as well versus bash scripts.

Comment (ammario) [reply main.go]: The problem is the bin is 90MB.

**Pull request merged**